synapse/tests/test_tools_aha.py

```diff
@@ -76,3 +82,71 @@ async def test_aha_easycert(self):
                 outp.expect('users/mallory@demo.net.key')
                 outp.expect('crt saved')
                 outp.expect('users/mallory@demo.net.crt')
+
+    async def test_aha_enroll(self):
+
+        with self.getTestDir() as dirn:
```

Comment:
could some of this aha boilerplate be replaced by getTestAhaProv ?

Reply:
getTestAhaProv sets `aha:urls` rather than using the default which is part of what i was trying to test here